tests/conftest.py

```diff
@@ -0,0 +1,64 @@
+from importlib.resources import files
+
+import pytest
+import responses
+
+from grafana_import.grafana import Grafana
+from grafana_import.util import grafana_settings, load_yaml_config
+from tests.util import mock_grafana_health, mock_grafana_search
+
+
+@pytest.fixture(scope="session", autouse=True)
+def niquests_patch_all():
+    """
+    Patch module namespace, pretend Niquests is Requests.
+    """
+    from sys import modules
+
+    import niquests
+    import urllib3
+
+    # Amalgamate the module namespace to make all modules aiming
+    # to use `requests`, in fact use `niquests` instead.
+    modules["requests"] = niquests
+    modules["requests.adapters"] = niquests.adapters
+    modules["requests.sessions"] = niquests.sessions
+    modules["requests.exceptions"] = niquests.exceptions
+    modules["requests.packages.urllib3"] = urllib3
```

Comment:
It might only be a nit, but I would like to highlight such procedures better on behalf of the [Niquests documentation](https://niquests.readthedocs.io/), in order to make the road less bumpy for people writing software tests for applications or libraries which are based on Niquests.

- https://github.com/jawah/niquests/issues/109

Now, I only need to find the time.

/cc @Ousret